content/_help/verify-your-identity/phone-number-and-phone-plan-in-your-name._en.md

```diff
@@ -1,22 +1,18 @@
 ---
```

Comment:
It may also be worth considering renaming the files to remove references to "phone plan in your name". I think you can rename them without any other changes being necessary.

e.g.

- `phone-number._en.md`
- `phone-number._es.md`
- `phone-number._fr.md`

Reply:
Good catch! Just committed this change